### DISCUSSION TITLE
How to use multiple directories to store data？

### DISCUSSION THREAD

**Discussion by sunwuyan** [longhorn/longhorn#8364] [General]
On each of my kubernetes nodes, there are two large-capacity Volume mounts, corresponding to solid-state drives and mechanical hard drives. I used the helm chart to install the longhorn cluster, which only uses /var/lib/longhorn by default to store data. How can I use both directories on each node?3q！

**Comment by mantissahz** [comment]
I think that will be like the section [add a disk](https://longhorn.io/docs/1.6.1/nodes-and-volumes/nodes/multidisk/#add-a-disk)
